Makefile

```diff
@@ -80,10 +81,8 @@ $(local_handler_manifest): deploy/operator.yaml
 	sed "s#REPLACE_IMAGE#$(LOCAL_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)#" \
 		deploy/operator.yaml > $@
 
-$(CLUSTER_DIR)/%: kubevirtci.version
-	rm -rf kubevirtci
-	git clone https://github.com/kubevirt/kubevirtci
-	cd kubevirtci && git checkout $$(cat  ../kubevirtci.version)
+$(CLUSTER_DIR)/%: $(install_kubevirtci)
```

Comment:
this should depend on kubevirtci as well, in case it is missing

Reply:
kubevirtci is part of the CLUSTER_DIR if it's not there make will call the dependencies.

CLUSTER_DIR = kubevirtci/cluster-up/